sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/RequestRetryTestFactory.java

```diff
@@ -372,17 +372,17 @@ private long calcPrimaryDelay(int tryNumber) {
 
         private OffsetDateTime calcUpperBound(OffsetDateTime start, int primaryTryNumber, boolean tryingPrimary) {
             if (tryingPrimary) {
-                return start.plus(calcPrimaryDelay(primaryTryNumber) * 1000 + 500, ChronoUnit.MILLIS);
+                return start.plus(calcPrimaryDelay(primaryTryNumber) * 1000 + 1000, ChronoUnit.MILLIS);
```

Comment:
What's the reason of the switching 500 to 1000?
Same as below. Any justification on this?